### PR TITLE
add a validation error for false template invitation email properties (server)

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
The translations currently arrive messed up, and it seems nice to return some validation error feedback here.

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
Ok, good to check for issues here. The reason why we got this error is another thing. I'll create an issue for the reason.

File: openslides/users/models.py [hunk 1]
```diff
@@ -232,10 +232,20 @@ def __missing__(self, key):
                 "password": self.default_password,
             }
         )
-        message = message.format(**message_format)
+        try:
+            message = message.format(**message_format)
+        except KeyError as e:
```
Comment (ostcar) [inline openslides/users/models.py]: Please use a longer variable name then `e`. For example `err`

File: openslides/users/models.py [hunk 2]
```diff
@@ -232,10 +232,20 @@ def __missing__(self, key):
...
+            raise ValidationError(
+                {"detail": f"Invalid property {e}."}
+            )
 
         subject_format = format_dict({"event_name": config["general_event_name"]})
-        subject = subject.format(**subject_format)
+        try:
+            subject = subject.format(**subject_format)
+        except KeyError as e:
```
Comment (ostcar) [inline openslides/users/models.py]: The same here

**Comment by MaximilianKrambach** [comment]
I fixed the two short variable names and rebased it all on the current master (this time the commit is not that changing that a rebase would confuse anyone)

**Pull request merged**